### PR TITLE
emission plots update

### PR DESCRIPTION
**Pull request by Bonface-Osoro**
Here is the updated plots.

In the first social cost plot; why did you make the y label as follows "ylab("Social Cost<br>(US$ Millions/t CO<sub>2</sub>eq)")" instead of how I named them using the bquote (y = bquote("Social Cost (US$ Millions/t CO"['2']~"eq)"))

### CODE REVIEW

File: vis/emissions.R [hunk 1]
```diff
@@ -49,13 +49,12 @@ individual_emissions$category = factor(
...
-##plot1 
-##########################
+##################################
+##plot1  climate change baseline##
+##################################
 df = individual_emissions %>%
   group_by(`Constellation`, category) %>%
-  summarize(mean = mean(`Climate Change - Global Warming Potential 100a`)
-  )
+  summarize(mean = `Climate Change - Global Warming Potential 100a`)
```
Comment (edwardoughton) [inline vis/emissions.R:57]: This isn't a mean value though - this is massively confusing to external reviewers. If something isn't a mean, do not call it a mean. If you need to call it something else, use "value". Or some other generic term.

File: vis/emissions.R [hunk 2]
```diff
@@ -103,17 +103,17 @@ climate_change <-
...
   )
 
 
-#####################################
-##plot2
-#####################################
+#######################################
+##plot2 climate change worst scenario##
+#######################################
 colnames(individual_emissions)
```
Comment (edwardoughton) [inline vis/emissions.R:113]: colnames() is residual/redundant - remove

**Review by edwardoughton — CHANGES_REQUESTED** [review]
Please see my comments:
- Do not call the value 'mean' if it's not a mean. This needs to be labelled logically. 
- Remove redundant functions e.g., colnames(). Only have code relevant to creating the plots/analysis. 
- Make sure all figures are being written out to saleos/vis/figures folder (so not saleos/data, figures are not data).

Please ask me if you have questions.

**Comment by Bonface-Osoro** [comment]
Updated the functions with docstring documentation

**Pull request merged**